### DISCUSSION TITLE
Open WebUI on Fly.io

### DISCUSSION THREAD

**Discussion by kylemclaren** [open-webui/open-webui#662] [Show and tell]
Hey, I wanted to get Ollama and Web UI up and running on [Fly GPUs](https://fly.io/gpu). It's working well so far on an Nvidia L40s! They're both running on the same Machine with "scale-to-zero"!

Check out the demo code here: https://github.com/fly-apps/ollama-open-webui

Edit: Fly GPUs are now GA 🎉

**Comment by justinh-rahb** [comment]
Nice! Might need to join that waitlist and check it out, been wanting to use L40S for a few other projects I've got in mind too. Thanks for sharing!

**Comment by justinh-rahb** [comment]
Just a heads-up @kylemclaren about the rename of the project, I'm sure people will love to continue getting the latest version on any available platform 😎 🚀

**Reply by kylemclaren** [reply]
Cool, updated the repo: https://github.com/fly-apps/ollama-open-webui
